### PR TITLE
more verbose README

### PR DESCRIPTION
**Pull request by Akira25**
Add some documentation on the structure and where to add stuff and so on.

### CODE REVIEW

**Comment by Akira25** [comment]
feel free to comment

File: README.md [hunk 1]
```diff
@@ -1,41 +1,121 @@
...
+#### model_files
+
+These files define how bbb-configs needs to handle different hardware models. This example shows a WDR4900:
+```sh
+---
+imagebuilder: "https://downloads.openwrt.org/releases/21.02-SNAPSHOT/targets/mpc85xx/p1010/openwrt-imagebuilder-21.02-SNAPSHOT-mpc85xx-p1010.Linux-x86_64.tar.xz"
+feed_version: "1.2.0-snapshot"
+imagebuilder_disable_signature_check: true      # don't check the signature-keys when installing packages ??
+
+switch_ports: 6             # What's that for?
```
Comment (PolynomialDivision) [inline README.md:66]: number of ethernet ports a device has + cpu

File: README.md [hunk 2]
```diff
@@ -1,41 +1,121 @@
...
+switch_int_port: 0          # What's that for?
```
Comment (PolynomialDivision) [inline README.md:67]: cpu port

File: README.md [hunk 3]
```diff
@@ -1,41 +1,121 @@
...
+switch_ignore_ports: []     # What's that for?
```
Comment (PolynomialDivision) [inline README.md:68]: sometimes, not all switch ports are usuable. for example the sxtsq 5 ac has "6" ports, but oly cpu port + port 5 is used (1 port device)

File: README.md [hunk 4]
```diff
@@ -1,41 +1,121 @@
...
+
+int_port: eth0              # What's an int-port ??
```
Comment (PolynomialDivision) [inline README.md:70]: what swconfig device, e.g. `eth0` or `eth1`

File: README.md [hunk 5]
```diff
@@ -1,41 +1,121 @@
...
+There are also files for the standard ssh keys and definitions for the wifi profiles.
+
+#### model_files
+
+These files define how bbb-configs needs to handle different hardware models. This example shows a WDR4900:
+```sh
+---
+imagebuilder: "https://downloads.openwrt.org/releases/21.02-SNAPSHOT/targets/mpc85xx/p1010/openwrt-imagebuilder-21.02-SNAPSHOT-mpc85xx-p1010.Linux-x86_64.tar.xz"
+feed_version: "1.2.0-snapshot"
+imagebuilder_disable_signature_check: true      # don't check the signature-keys when installing packages ??
```
Comment (PolynomialDivision) [inline README.md:64]: @Akira25 Are you checking keys? I think we currently only rely on https when downloading packages. However, nice feature improvement.

**Comment by PolynomialDivision** [comment]
nice thanks for documentation! maybe adding some dsa device would be also nice.

File: README.md (hunk 5 above)
Comment (Akira25) [reply README.md:64]: Of course we check the keys on falter. OpenWrt-21.02 imagebuilders don't work without it. Why does ansible deactivate this? Having the hassle with the keys and encryption and not using them for security seems pointless to me.

**Comment by Akira25** [comment]
> nice thanks for documentation! maybe adding some dsa device would be also nice.

If you give me a hint, where to find such a configuration, I could do that.

File: README.md (hunk 5 above)
Comment (spolack) [reply README.md:64]: Its not yet implemented
https://github.com/Freifunk-Spalter/bbb-configs/issues/19

**Comment by spolack** [comment]
> If you give me a hint, where to find such a configuration, I could do that.

https://github.com/Freifunk-Spalter/bbb-configs/blob/master/group_vars/model_ubnt_edgerouter_x_sfp.yml

There you can also see how poe ports are being described. On https://github.com/Freifunk-Spalter/bbb-configs/blob/ab19e96532aa6d0ab58cd1425ca144288df5f034/host_vars/koepi-trailer1/base.yml#L8 you can see, how to enable individual POE ports.

**Comment by Akira25** [comment]
@spolack @PolynomialDivision I think this should be ready now.

File: README.md [hunk 6]
```diff
@@ -1,41 +1,196 @@
...
+location_nice: roof top, Example-Street 24, 10573 Berlin        # any string that describes your location. Could be an address or something
+latitude: 52.484948320
+longitude: 13.443380903
+```
+
+##### networks.yml
+
+This file is the most important one of your setup. It defines IP-Addresses, WIFI-Properties, Mesh and so on. Lets have a (shortened) example of a `networks.yml` from the magda-location:
+
+```sh
```
Comment (spolack) [inline README.md:114]: That should be yaml i guess. I never use that infostring, not sure if you did this on purpose

File: README.md [hunk 7]
```diff
@@ -1,41 +1,196 @@
...
+---
+
+location: sama                  # locations short name, like def'd in location-directories
+role: corerouter                # devices role. Could either be 'corerouter', 'ap' or 'uplink_gateway'
+target: "ipq40xx/generic"       # target of router model
+model: "avm_fritzbox-7530"      # model name like written in the corresponding file name in group_vars/
+
+wireless_profile: freifunk_default      # activates wifi with freifunk-default-settings on this device
```
Comment (spolack) [inline README.md:174]: .. by overriding default wireless profile for corerouters, which is the profile `disable`

**Review by spolack — APPROVED** [review]
Thank you for your contribution. very appreciated!

File: README.md (hunk 6 above)
Comment (Akira25) [reply README.md]: I did it on purpose.vscode complained, that markdown standard requires that.

File: README.md (hunk 7 above)
Comment (Akira25) [reply README.md]: added in new commit-version

**Comment by Akira25** [comment]
@spolack Threres one approval already. Can we merge this?

**Comment by spolack** [comment]
Sure!

**Pull request merged**